### PR TITLE
Flex_01

### PR DESCRIPTION
**Pull request by Prapti-Patel**
# Pull request

## Implement

- [ ] Update pull request (title, assignee, labels, projects, implement, measure)
- [ ] Update issue numbers here which will be fixed by this PR | Fixes #
- [ ] Update labels for clarity & PR workflow
- [ ] Add/Update comments in code
- [ ] Self code review as per this [guide](https://google.github.io/eng-practices/review/)
- [ ] Add/Update change logs, docs and release notes

## Measure

- [ ] What: Pull Request Title | Why: PR Detail | How: DIM | When: Timeline | Who: Assignee & Reviewer
- [ ] Review change logs, docs & release notes
- [ ] After completing all subtasks validate and make PR ready to merge

### CODE REVIEW

**Comment by Prapti-Patel** [comment]
@beingminimal Please review.

File: individual-projects/2023/Prapti-Patel-190770107179/Flex_01/style.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+nav ul {
+    margin: 0;
+    height: 60px;
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: absolute;
```
Comment (KunalChudasama) [inline individual-projects/2023/Prapti-Patel-190770107179/Flex_01/style.css:23]: Why you have used position....?

File: individual-projects/2023/Prapti-Patel-190770107179/Flex_01/style.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    background-color:beige;
+
+}
+
+nav ul li {
+    display: inline;
```
Comment (KunalChudasama) [inline individual-projects/2023/Prapti-Patel-190770107179/Flex_01/style.css:29]: why you have used display inline here...??

File: individual-projects/2023/Prapti-Patel-190770107179/Flex_01/style.css [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+  h3{
+    font-size: 15px;
+  }
+.btn{
+    width: 100%;
+    margin-top: 70%;
+    padding: 8px;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
```
Comment (KunalChudasama) [inline individual-projects/2023/Prapti-Patel-190770107179/Flex_01/style.css:78]: what is the need of Align-items:flex-end...?

**Review by KunalChudasama — CHANGES_REQUESTED** [review]
Changes Required

**Comment by KunalChudasama** [comment]
@Prapti-Patel How many examples of flexbox you have completed..?

**Comment by Prapti-Patel** [comment]
@KunalChudasama I have completed two examples of flexbox

**Comment by KunalChudasama** [comment]
@Prapti-Patel what is the file name of the second example.....???

**Comment by Prapti-Patel** [comment]
@KunalChudasama Flex_02 
But this example approved by Amoli ma'am

**Review by KunalChudasama — APPROVED** [review]

**Pull request merged**